flamingo/flamingo.h

```diff
@@ -399,6 +399,24 @@ int flamingo_run(flamingo_t* flamingo);
  */
 flamingo_var_t* flamingo_find_var(flamingo_t* flamingo, char const* key, size_t key_size);
 
+/**
+ * Increment the reference count of a value.
+ *
+ * This should be used when the external program needs to hold on to the value.
+ *
+ * @param val The value to increment the reference of.
+ */
+void flamingo_val_incref(flamingo_val_t* val);
+
+/**
+ * Decrement the reference count of a value.
+ *
+ * This should be used when the external program needs to let go of a value it previously referenced with {@link flamingo_val_incref}.
```

Comment:
The docs say flamingo_val_decref is just for things you've "incref’d," but you actually need it for any value made with the flamingo_val_make_* functions too, since they start with a reference count of 1.